Aardvark.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
   s.name     = 'Aardvark'
-  s.version  = '2.0.0'
+  s.version  = '2.0.1'
   s.license  = 'Apache License, Version 2.0'
   s.summary  = 'Aardvark is a library that makes it dead simple to create actionable bug reports.'
   s.homepage = 'https://github.com/square/Aardvark'
   s.authors  = 'Square'
   s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => s.version }
-  s.ios.deployment_target = '6.0'
+  s.ios.deployment_target = '8.0'
```

Comment:
(not a blocker since this is a Square-specific branch) technically this is a breaking change, so the version bump should be 3.0.0.

Reply:
Not necessarily Square specific, but for anyone who can't use dynamic frameworks with CocoaPods. Fortunately, CP handles deployment targets correctly based on the Podfile's platform version.

Reply:
I defer to your judgment here then :)